examples/jvm/java/src/main/java/io/fission/HelloWorld.java

```diff
@@ -0,0 +1,31 @@
+package io.fission;
+
+import java.util.LinkedHashMap;
+
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.fission.FissionFunction;
+import io.fission.FissionContext;
+
+public class HelloWorld implements FissionFunction<RequestEntity, FissionContext> {
```

Comment:
A bit of a design question, can users get around not having to import the entire springframework? If not, you might want to consider one of two options:
- rename the environment to `java` or `java-spring` to indicate that it is specific to spring.
- also provide an option for users to use `RequestEntity` and `ResponseEntity`-like objects that are in the `io.fission` package. (Such that users can choose to only import the `io.fission` package to create their function)

Reply:
@erwinvaneyk If we have to provide a Request/Response object as part of Fission library - that would be a lot of work. Using the spring framework is like using Flask framework in Python environment - we choose a fairly commonly used web framework. I am ok with renaming environment to java-spring as that clearly indicates the decision.

Reply:
@vishal-biyani I understand :) Renaming the environment to `java-spring` is more self-documenting to and sets the expectations for users. It also gives us the option to later add support for a different (whichever is most popular by then :P) framework or plain java without having to maintain and explain the caveat that `java` == `spring`. 👍